src/lxc/confile.c

```diff
@@ -3111,7 +3111,9 @@ bool lxc_config_define_load(struct lxc_list *defines, struct lxc_container *c)
 			break;
 	}
 
+#ifndef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
 	lxc_config_define_free(defines);
+#endif /* !FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION */
```

Comment:
I have to say I'm not sure I like this kludge :-) but it seems to be the easiest way to keep all the keys `lxc_config_define_load` has just processed to make it possible to iterate over them to make sure `get_config_item` can properly extract their values (previously set with `set_config_item`). Another option would be to copy the keys somewhere between lxc_config_define_add and lxc_config_define_load but it seems to slow down the fuzzer unnecessarily. @brauner I'd appreciate it if you could take a look and point me in the right direction :-)

Reply:
The idea is to cover the "get_*" functions from `config_jump_table`. It should help to trigger scenarios where `set_config_*` followed by `get_config_*` can cause unexpected side effects like half-initialiazied structures and so on

Reply:
Hm, I think for now that your hack is fine. :)